app/assets/stylesheets/common/_markus.scss

```diff
@@ -303,6 +303,12 @@ button,
   }
 }
 
+#assignment-delete {
```

Comment:
Okay this is better, but (1) make it a class, and (2) call the class `danger-button`, which is more general.